common/views/components/IIIFViewer/IIIFViewer.js

```diff
@@ -292,8 +295,27 @@ const IIIFViewerComponent = ({
     : null;
 
   // Download info from manifest
+  const urlTemplateMain = mainImageService['@id']
+    ? iiifImageTemplate(mainImageService['@id'])
+    : null;
+  const largeImage = urlTemplateMain && {
+    '@id': urlTemplateMain({ size: 'full' }),
+    format: 'image/jpeg',
+    label: 'This image (Full size)',
+  };
+  const smallImage = urlTemplateMain && {
+    '@id': urlTemplateMain({ size: '760,' }),
+    format: 'image/jpeg',
+    label: 'This image (760 pixels)',
+  };
   const iiifPresentationDownloadOptions =
-    (manifest && getDownloadOptionsFromManifest(manifest)) || [];
+    (manifest &&
+      [
+        ...getDownloadOptionsFromManifest(manifest),
```

Comment:
should this be renamed now that `largeImage` and `smallImage` are also download options from the manifest?

Reply:
I still think it makes sense, there are just more options now.

Reply:
Oh I get you. Yeah, makes sense.